autonomy/cli/service.py

```diff
@@ -91,13 +99,15 @@ def activate(
     help="Agent ID",
 )
 @key_path_decorator
+@hwi_flag
 @password_decorator
 def register(  # pylint: disable=too-many-arguments
     ctx: Context,
     service_id: int,
     instances: List[str],
     agent_ids: List[int],
-    keys: Path,
+    key: Path,
```

Comment:
This backwards incompatible change needs to go into upgrade notes! What's the reason we had "keys" before? Typo?

Reply:
Yes, just badly chosen variable name. Although this is a breaking change it's internal since this functions is not supposed to be used by the end users.